docs/src/snoopr.md

```diff
@@ -128,7 +137,7 @@ In this example, there were four different methods that triggered invalidations,
 You can see that collectively more than a thousand independent compiled methods needed to be invalidated; indeed, the last
 entry alone invalidates 1027 method instances:
 
-```
+```julia
```

Comment:
I thought this is the default for Documenter (not for GitHub), but now I can't find evidence of that.

Reply:
I guess it is better to stay on the safe side 😁